src/dbl.ml

```diff
@@ -28,7 +28,15 @@ let cmd_args_options = Arg.align
 
     "-verbose-internal-errors",
     Arg.Set InterpLib.InternalError.verbose,
-    " Make internal errors more verbose (for debugging only)"
+    " Make internal errors more verbose (for debugging only)";
+
+    "-I",
+    Arg.String (fun p -> Config.lib_search_dirs := !Config.lib_search_dirs @ [p]),
+    " Standard library paths";
```

Comment:
This option takes just one path, which I don't think is clear from the description. I'd also prefer it to be phrased in terms of an action that the option will perform, like "add a directory ...", to be consistent with the other help messages.